packages/react-examples/src/react-badge/Badge/Badge.stories.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
-import { Badge, BadgeProps } from '@fluentui/react-badge';
+import { Badge } from '@fluentui/react-badge';
 
-const BadgeAppearanceExamples = (props: BadgeProps) => (
+const BadgeAppearanceExamples = () => (
```

Comment:
Following up from the previous review (https://github.com/microsoft/fluentui/pull/16868#discussion_r579466170): why not export this directly, instead of having a wrapper component below that you export? I'm not sure I understood what you meant about boilerplate.